.github/workflows/documentation_deploy.yml

```diff
@@ -3,6 +3,8 @@ name: '[Documentation] Deploy Site'
 # Adapted from https://docusaurus.io/docs/deployment
 
 on:
+  push:
+    branches: ["develop"]
```

Comment:
Would it be useful to use a [path filter](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#onpushpull_requestpull_request_targetpathspaths-ignore) here so that this only runs when there are changes in `VAMobile/documentation`?

Reply:
yeah good call out, lemme update that

Reply:
added, hard to test, but I think I got the syntax